plugins/hls-refactor-plugin/src/Development/IDE/GHC/ExactPrint.hs

```diff
@@ -623,6 +653,14 @@ eqSrcSpanA l r = leftmost_smallest l r == EQ
 #endif
 
 #if MIN_VERSION_ghc(9,2,0)
+
+spanContainsRange :: SrcSpan -> Range -> Bool
```

Comment:
Maybe you can use [`realSrcSpanToRange`](https://github.com/haskell/haskell-language-server/blob/163ebfe9d02d5aa9f2c83ef8d9db2e91cde31032/ghcide/src/Development/IDE/GHC/Error.hs#L76) and [`isSubrangeOf`](https://github.com/haskell/lsp/blob/a41954e356117d8f5cb64b20e1a9d32e8fc6a884/lsp-types/src/Language/LSP/Types/Location.hs#L89) ?

Reply:
I forgot to respond. This worked perfectly thanks :)